src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -220,14 +220,14 @@ static std::tuple<std::shared_ptr<ov::Node>, std::shared_ptr<ov::Node>> gpt_oss_
 
     auto add = wrap_type<v1::Add>({q_idx, offset});
     auto opt_conv_2 = pattern::optional<v0::Convert>(add);
-    auto greater = pattern::wrap_type<v1::Greater>({kv_idx_opt_conv_1, opt_conv_2});
+    auto greater = pattern::wrap_type<v1::Greater>({any_input(), opt_conv_2});
```

Comment:
The intention of the original version of the code was to work with "any_input" as well, this is why both of the "Converts" are marked optional. 
With the change, related variables above are not used (so could be removed):
```
   auto kv_idx = pattern::any_input();

    auto kv_idx_opt_conv_0 = pattern::optional<v0::Convert>();
    auto kv_idx_opt_conv_1 = pattern::optional<v0::Convert>(kv_idx_opt_conv_0);
```

```suggestion
    auto greater = pattern::wrap_type<v1::Greater>({kv_idx, opt_conv_2});
```